scylla_pg_core/Cargo.toml

```diff
@@ -1,14 +1,13 @@
 [package]
 name = "scylla_pg_core"
-version = "0.1.22"
+version = "0.1.23"
```

Comment:
There are some packages with older versions. Shouldn't we update `cargo.toml` version to `0.1.23` in all the packages?